### DISCUSSION TITLE
Anyone Else? - unable to switch between output settings?

### DISCUSSION THREAD

**Discussion by takethecan0lis** [bitgapp/eqMac#613] [Q&A]
Hi All,  when EQMac works it really works well but there's an issue I'm having where I can't seem to apply the EQ settings to various selected outputs.  As an example, I have a Topping DX3 connected via USB to my mac, and while I can select it in eqMac, and I can adjust volume settings, EQ and Spatial audio setting only seem to work intermittently. As a work around, I need to constantly reboot eqMac as well as my entire laptop to get the settings to take effect but that doesn't seem to work every time either.  Is this a known issue?  I'm unsure of what common keywords are best to search for to see if it's been reported before.  I don't want to double report and get banished.  

App V1.3.2 Driver V1.3.2 UI V 3.1.1

Edited: Figured out what's happening.... If I select my output in Tidal then it somehow bypasses eqMac, but if I select my output in OSX it works.  The hassle their is that if I want to apply different EQ settings to music vs. Zoom calls I have to quickly open up eqMac to make the changes.

Edit 2:Ok 2 days have passed and I just realized I've been doing it all wrong..... I realized that in OSX's volume preference I can assign the output to eqMac and then change outputs in the eqMAC UI (mind blown!!).  That's super powerful especially combined with my new found hobby of modifying mechanical keyboards (double geek).   Does anyone know if there's a way for me to map a keystroke to select volume outputs?  The use case is that I listen to music via a USB DAC > Stereo, but I take my video calls on either a headphone or my macs native speakers.  I don't always use the same video call service (Google, Zoom, Teams, etc...).

Another question... Can I save unique audio settings for each output in eqMac?  For instance I don't want any spatial effects for video calls, but I use them for listening to music.  Does eqMac remember output specific settings?

Seriously... eqMac just became so much cooler for me.

**Comment by nodeful** [comment]
Hey there, the developer of eqMac here.
So your post got me a little confused, you should be able to switch the device from either eqMac UI or the System UI, that should not matter. But other people have reported that there's some issues with Tidal reported in this issue: #604 
And also I've been getting emails with people reporting issues with Tidal, so I'll have to take a more thorough look into that once I have some time. 
You should be able to also adjust volume from eqMac or System UI and it should really sync everything up nicely, so I'm not sure what's going on, if you could record a video from your phone showcasing the issue you're experiencing I think that would be very helpful.

Regarding second part of your message, I think what you're talking about is 2 things:
1. Automatic switching of EQ presents when you're changing Output device, this is a long standing "Feature Request": #231 and quite high on my priority list so should be implemented soon.
2. You want to control eqMac with some sort of "hotkeys" which is a feature also requested a while ago: #338 and also it's part of the "coming feature voting" on the website: https://eqmac.app#coming-soon
![image](https://user-images.githubusercontent.com/8472525/147009501-165fd6e4-0be3-4f23-b5c1-5239be18625a.png)
